act/headings.md

```diff
@@ -0,0 +1,137 @@
+# Headings
+
+## Rule Identifier
+
+gel-foundations-headings
+
+## Rule Description
+
+Headings provide a semantic and visual hierarchical structure to a document.
+
+This rule tests that pages have a heading structure meets the requirement of BBC GEL Headings.
+
+## Rule Type
+
+atomic
+
+## Accessibility Requirements Mapping
+
+- [BBC GEL Headings](https://bbc.github.io/gel/foundations/headings/)
+  - Required for conformance to BBC GEL
+  - Outcome mapping:
+    - Any failed outcomes: success criterion is not satisfied
+    - All passed outcomes: success criterion needs further testing
+    - An inapplicable outcome: success criterion needs further testing
+
+## Input Aspects
+
+DOM Tree
+
+## Applicability
+
+This rule applies to any HTML page with a heading structure.
```

Comment:
It might be useful to describe, or give examples later, of cases where it _wouldn't_ apply?

Reply:
Good point. I don't this approach applies to many others, but I'll look around the BBC for an example that I can use.